docs/metadata.md

```diff
@@ -0,0 +1,73 @@
+
+## Dispute
+
+Structure from the graph
+
+```javascript
+type Dispute @entity {
+  id: ID!
+  ...
+  subject: Arbitrable!
+  metadata: String!
+  ...
+}
+```
+
+```javascript
+type Arbitrable @entity {
+  id: ID!
+  disputes: [Dispute!] @derivedFrom(field: "subject")
+}
+```
+
+In the frontend we try to parse the `metadata` as a json object.
+
+### metadata - Json object
+
+We are going to try to find the following keys:
+
+1. `description`
+2. `agreementText`
+3. `defendant`
+4. `plaintiff`
+
+#### metadata - String
+
+In case that the metadata is just a plain string we are going to use it as the dispute description and the dispute creator `subject.id` will be considered as the plaintiff
```

Comment:
```suggestion
In case `metadata` is a plain string, we assume it to be the dispute's description and set the dispute creator (`subject.id`) as the plaintiff. No agreement link will be available.
```